### ISSUE TITLE
Issue with tf.arg_max in inpainting_ops.py

### ISSUE THREAD

**Issue by avidkenil** [JiahuiYu/generative_inpainting#33]
I tried to run the test script on the pre-trained models for the image net task on cpu on Mac OS X Sierra with Tensorflow 1.7.0 and it gave me an error that there is not output_type argument for tf.arg_max in inpainting_ops.py on line 302, I was able to fix that bug using this line. 

`offset = tf.cast(tf.argmax(yi, axis=3),tf.int32)`

Just bringing this to notice if anyone faces similar bug in the future and would be better to cast it for preventing the bug from causing.

**Comment by JiahuiYu** [comment]
Thanks for your information. I've been working with tf 1.7.0 on linux, there is no such problem. Also in tensorflow API, the arguments still exist. Not sure what is going no with Tensorflow 1.7.0 on osx. Anyway, thanks for your info.
